keep/api/routes/alerts.py

```diff
@@ -607,7 +608,12 @@ async def receive_generic_event(
     for _alert in alert:
         # if not source, set it to keep
         if not _alert.source:
-            _alert.source = ["keep"]
+            if authenticated_entity.api_key_name:
+                _alert.source = [authenticated_entity.api_key_name]
```

Comment:
two things - 
1. The source should be still "keep". Add another column to alerts that will reference to the API key
2. usedApiKey is redundant - just use one column "api_key_ref" so we have the reference to the api key

Reply:
fixed e1272835edd0127b3517106ed576607ca4f1c770